### DISCUSSION TITLE
blogs/view/use-your-brother-printer-on-lan-by-raspberry-pi

### DISCUSSION THREAD

**Discussion by giscus** [biplobsd/biplobsd.github.io#17] [Announcements]
# blogs/view/use-your-brother-printer-on-lan-by-raspberry-pi

Make your Brother printer a print server using the Raspberry Pi. In this blog, I will share my setup... | Blogs | Biplob Sutradhar

https://biplobsd.me/blogs/view/use-your-brother-printer-on-lan-by-raspberry-pi.md

**Comment by prathamjoshi0909** [comment]
hello. i have te same model and i did the steps but printer is not showing in cups. any solution??

**Reply by biplobsd** [reply]
@itachi0909 Tried the [Driver Install Tool](https://support.brother.com/g/b/downloadlist.aspx?c=us_ot&lang=en&prod=dcpt220_all&os=128#SelectLanguageType-625_0_1) ?
